package.json

```diff
@@ -63,24 +75,42 @@
     "webpack-cli": "^3.3.12"
   },
   "main": "./dist/ext/extension.js",
-  "browser": "./dist/web/extension.js",
   "activationEvents": [
+    "onLanguage:dfdl",
     "onDebugResolve:dfdl",
     "onDebugDynamicConfigurations:dfdl",
     "onCommand:extension.dfdl-debug.getProgramName",
     "onCommand:extension.dfdl-debug.getDataName",
     "onCommand:extension.dfdl-debug.runEditorContents",
     "onCommand:extension.dfdl-debug.debugEditorContents",
-    "onCommand:launch.config"
+    "onCommand:launch.config",
+    "onCommand:data.edit",
+    "onCommand:omega_edit.version",
+    "onCommand:toggle.experimental"
   ],
   "workspaceTrust": {
     "request": "never"
   },
   "contributes": {
-    "snippets": [
+    "languages": [
       {
-        "language": "xml",
-        "path": "./snippets/dfdl.json"
+        "id": "dfdl",
+        "aliases": [
+          "dfdl",
+          "dfdl"
+        ],
+        "extensions": [
+          ".xsd",
+          ".xml"
```

Comment:
Yes that would be best. When we were testing we only allowed for `.dfdl.xsd` however in some schemas they were using just `.xsd` so the language extension wouldn't be enabled as well as breakpoints could not be added to these files causing issues. So create a standard `.dfdl.xsd` and `dfdl.xml` (should infoset have this extension?) would be good.

Reply:
I think infosets should just be treated like normal XML files.

If you wanted to get really fancy, you could have autocomplete editing of infosets generated from DFDL schemas. I mentioned in another comment that there are extensions out there that seem to be able to do this already with XSD's, and since DFDL schemas are just normal XSD as far as infosets are aware, something like that should work in theory.

Reply:
> When we were testing we only allowed for .dfdl.xsd however in some schemas they were using just .xsd

This is probably fine. I think it's reasonable to enforce that this only works with .dfdl.xsd files. Alternatively (if possible) you could have a setting that says what extensions to treat as DFDL files. It would probably only want to default to dfdl.xsd, but people could add .xml/xsd if they wanted. I'm not sure if vscode supports that or if extensions must be hardcoded in though.

Reply:
Okay I will remove `.xml` and update `.xsd` to be `.dfdl.xsd` sound good?

Reply:
Sounds good to me.